components/landing/TopCollectionsItem.vue

```diff
@@ -12,45 +12,127 @@
             rounded
             :src="collection.image || '/placeholder.webp'" />
         </div>
-        <div class="px-2">
+        <div class="px-2 is-flex is-flex-grow-1 is-flex-direction-column">
           <div class="has-text-weight-bold">
             {{ collection.name | truncateStr(12) }}
           </div>
-          <div class="has-text-grey is-size-7">
-            <div v-if="collection.floorPrice">
-              Floor: <CommonTokenMoney :value="collection.floorPrice" inline />
+          <div class="is-flex is-justify-content-space-between nowrap">
+            <div>
+              <div v-if="collection.floorPrice">
+                Floor:
+                <CommonTokenMoney :value="collection.floorPrice" inline />
+              </div>
+              <div v-else>---</div>
+            </div>
+            <div class="is-uppercase has-text-grey pl-2">
+              {{ urlPrefix }}
             </div>
-            <div v-else>---</div>
           </div>
         </div>
       </div>
-      <div class="is-pulled-right has-text-right px-4">
-        <div>
-          <CommonTokenMoney :value="collection.volume" inline />
-        </div>
-        <div class="has-text-grey is-size-7">
-          avg. <CommonTokenMoney :value="collection.averagePrice" inline />
+      <div class="is-pulled-right has-text-right px-2">
+        <div class="is-flex">
+          <div>
+            <div>
+              <CommonTokenMoney :value="volume" inline />
+            </div>
+            <div>
+              <BasicMoney :value="usdValue" inline :unit="'USD'" />
+            </div>
+          </div>
+          <div
+            class="is-flex is-justify-content-center is-align-items-center pl-2 is-size-5">
+            <p :class="color">
+              {{ diffPercent.str }}
+            </p>
+          </div>
         </div>
       </div>
     </div>
   </nuxt-link>
 </template>
 
-<script lang="ts" setup>
+<script lang="ts">
```

Comment:
Why did you rewrite it nuxt2 way? 
we try to migrate slowly to nuxt3/vue3 (ref #2690)

Reply:
Honestly... I'm new to Vue , coming from react
I needed access to the store and could not find any example in the code base where it is done NOT using this.$store
so i figured it needs to be a class component

please enlighten me on how to do that properly

Reply:
You can do something like `const { $store } = useNuxtApp()` to acces Nuxt store